src/app/app.component.html

```diff
@@ -1,3 +1,5 @@
 <h1>
   {{title}}
 </h1>
+<router-outlet></router-outlet>
+<places-list></places-list>
```

Comment:
Also can you please add simple links:
list of places, and pages with few details.